pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -404,6 +410,17 @@ func manageControllerManagerKubeconfig(ctx context.Context, client corev1client.
 	return resourceapply.ApplyConfigMap(client, recorder, requiredCM)
 }
 
+func manageRecycler(ctx context.Context, configMapsGetter corev1client.ConfigMapsGetter, recorder events.Recorder, imagePullSpec string) (*corev1.ConfigMap, bool, error) {
```

Comment:
keep a storage team indication and name here.

Reply:
done